cmd/iot-agent/main.go

```diff
@@ -12,12 +12,15 @@ import (
 
 	"github.com/DataDog/datadog-agent/cmd/agent/app"
 	"github.com/DataDog/datadog-agent/pkg/util/flavor"
+	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 func main() {
 	// Set the flavor
 	flavor.SetFlavor(flavor.IotAgent)
+	// Change the iot_host default
+	config.Datadog.SetDefault("iot_host", true)
```

Comment:
https://github.com/DataDog/datadog-agent/blob/e081bed84866686eeab56a98da5ff9b4b9f03ded/cmd/iot-agent/main_windows.go#L26 needs an update as well. Or `SetDefault` could continue to live in the `flavor` pkg, which I think would make more sense.

Reply:
Complete nit but while we're looking at this: `flavor.SetFlavor` is redundat.